admin/data/sql/migrations/132.undo.drop-pin-gen-pupil-view.sql

```diff
@@ -0,0 +1,95 @@
+CREATE VIEW [mtc_admin].vewPupilsEligibleForPinGeneration AS
```

Comment:
does this overlap with 125? appears to be the same view

Reply:
125 drops the view so that 126 can update it

Reply:
Migrations 125 and 126 deleted